test/index.spec.js

```diff
@@ -511,6 +513,71 @@ describe('discipl-law-reg', () => {
       })
     })
 
+    it('should be able to take an action where the object originates from another action - LERARENBEURS', async () => {
+      let core = lawReg.getAbundanceService().getCoreAPI()
+
+      let lawmakerSsid = await core.newSsid('ephemeral')
+      await core.allow(lawmakerSsid)
+
+      let belanghebbendeSsid = await core.newSsid('ephemeral')
+      await core.allow(belanghebbendeSsid)
+      let bestuursorgaanSsid = await core.newSsid('ephemeral')
+      await core.allow(bestuursorgaanSsid)
+
+      let modelLink = await lawReg.publish(lawmakerSsid, { ...lb, 'model': 'LB' }, {
+        '[persoon wiens belang rechtstreeks bij een besluit is betrokken]':
+          'IS:' + belanghebbendeSsid.did
+      })
+
+      let retrievedModel = await core.get(modelLink)
+
+      let needSsid = await core.newSsid('ephemeral')
+
+      await core.allow(needSsid)
+      let needLink = await core.claim(needSsid, {
+        'need': {
+          'act': '<<indienen verzoek een besluit te nemen>>',
+          'DISCIPL_FLINT_MODEL_LINK': modelLink
+        }
+      })
+
+      let belanghebbendeFactresolver = (fact) => {
+        if (typeof fact === 'string') {
+          return fact === '[verzoek een besluit te nemen]' ||
+            // Interested party
+            fact === '[orgaan]' || fact === '[rechtspersoon die krachtens publiekrecht is ingesteld]'
+        }
+        return false
+      }
+
+      let actionLink = await lawReg.take(belanghebbendeSsid, needLink, '<<indienen verzoek een besluit te nemen>>', belanghebbendeFactresolver)
+
+      let bestuursorgaanFactresolver = (fact) => {
+        if (typeof fact === 'string') {
+          return fact === '[persoon wiens belang rechtstreeks bij een besluit is betrokken]' ||
+            fact === '[orgaan]' || fact === '[rechtspersoon die krachtens publiekrecht is ingesteld]' ||
```

Comment:
Since the facts `[orgaan]` and `[rechtspersoon die krachtens publiekrecht is ingesteld]` are about the identity of the actor, I think it makes sense to inject them with an IS: + bestuursorgaanDid, like we do in the other test with `[wetgevende macht]` and then remove them here